etsy/_core.py

```diff
@@ -288,7 +290,7 @@ def _get(self, http_method, url, **kwargs):
 
         if http_method == 'GET':
             url = '%s%s?%s' % (self.api_url, url, urlencode(kwargs))
-            body = ''
+            body = ''.encode('utf-8')
```

Comment:
what about the following else conditions?
Make it outside of if condition
and use isinstance to check it's already bytes or not

Reply:
@Anmol-Gulati We are directly assigning empty string then whats the need to check whether it is byte of not

Reply:
What was the error before this patch? Can you post stacktrace or something?